tools/benchmarks/classification/dist_train_linear.sh

```diff
@@ -8,11 +8,19 @@ PRETRAIN=$2  # pretrained model
 PY_ARGS=${@:3}
 GPUS=${GPUS:-8}  # When changing GPUS, please also change imgs_per_gpu in the config file accordingly to ensure the total batch size is 256.
 PORT=${PORT:-29500}
+NNODES=${NNODES:-1}
+NODE_RANK=${NODE_RANK:-0}
+MASTER_ADDR=${MASTER_ADDR:-'127.0.0.1'}
 
 # set work_dir according to config path and pretrained model to distinguish different models
 WORK_DIR="$(echo ${CFG%.*} | sed -e "s/configs/work_dirs/g")/$(echo $PRETRAIN | rev | cut -d/ -f 1 | rev)"
 
-python -m torch.distributed.launch --nproc_per_node=$GPUS --master_port=$PORT \
+python -m torch.distributed.launch \
+    --nproc_per_node=$GPUS \
```

Comment:
Such a modification should be updated in another PR, which does not related to MAE.

Reply:
Thank you for your valuable comments. The same problem as the modification in `gitignore` and I will fix it as soon as possible.